container/build.sh

```diff
@@ -146,11 +154,22 @@ main () {
     # This is because in some systems the BUILD files under /third_party (after git clone)
     # will be with permission 640 and the build will fail in Container Builder.
     find ${PROJECT_ROOT}/third_party -type f -print0 | xargs -0 chmod 644
-    # Start Google Cloud Container Builder
-    gcloud container builds submit . \
-      --config=${PROJECT_ROOT}/container/cloudbuild.yaml \
-      --substitutions _PROJECT=${PROJECT},_DISTRO=${DISTRO},_CONTAINER=${CONTAINER},_TAG=${TAG},_DIR=${DIR},_BUCKET=${BUCKET} \
-      ${ASYNC}
+
+    if [[ "$BUCKET" != "" ]]; then
+      # Start Google Cloud Container Builder
+      gcloud container builds submit . \
+        --config=${PROJECT_ROOT}/container/cloudbuild.yaml \
+        --substitutions _PROJECT=${PROJECT},_CONTAINER=${CONTAINER},_TAG=${TAG},_DIR=${DIR},_BUCKET=${BUCKET} \
+        --machine-type=n1-highcpu-32 \
+        ${ASYNC}
+    else
+      # Start Google Cloud Container Builder
+      gcloud container builds submit . \
+        --config=${PROJECT_ROOT}/container/cloudbuild_no_bucket.yaml \
```

Comment:
make the conditional only define the value of --config, and use the same call after the conditional